### DISCUSSION TITLE
New season, New to GitHub 🌟 [New to GitHub Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#33856] [New to GitHub]
With the changing of the season, September often brings the start of a new academic year for many as well which means even more folks are new to GitHub. Whether you’re in school 🎒 or not, it is always a good time to put on your learning hat 👒🎩🧢 and try something new in GitHub! 

🌟**Community Superstars**: A round of applause for these community citizens 👏
* @halldorr: For asking an extremely clear and concise [question](https://github.com/orgs/community/discussions/32354) on branches and Pull Requests ↪️
* @Franklin-Ngueiwoh: with the self-solve 😺, kudos for troubleshooting regenerating your PAT yourself and thanks for coming back to share the [solution](https://github.com/orgs/community/discussions/33049). 
* @averagebit: For yet another [great answer](https://github.com/orgs/community/discussions/32263), they included links to documentation and even provided an example 🤩. 

👀 **Around Community on Discussions**: The latest and greatest happenings in our Community 
* **Take our Survey**: Tell us what you think of github.com/community in a [brief survey](https://github.com/community/community/discussions/33536) **before October 4th** so we can continue to improve your community experience 🌞.
* **GitHub Skill** 🆙: @ossanna16 is posting weekly opportunities for you to expand your GitHub Skills, join [this week’s](https://github.com/orgs/community/discussions/33586) on Pull Requests! 

📚**Good Reads for Beginners**: Some old, some new, all great resources to take along with you on your GitHub journey 🗺️
* **GitHub Protips**: Improve your contributions, triage your issues efficiently, and more with @bdougie’s [protips](https://github.blog/2020-04-30-github-protips-tips-tricks-hacks-and-secrets-from-brian-douglas/). 
* 🆕 **How I Got Started with GitHub**: Read about how I learned GitHub and the resources I used in my [blog](https://github.com/orgs/community/discussions/32792). 
* **Career Tips for Beginner Developers**: @kelseybernius gives [advice](https://github.blog/2022-03-29-career-tips-for-beginner-developers/) on fundamentals, choosing a programming language, and beyond 🚀. 

Enjoy the reading, learning, and doing of all things GitHub! Stuck on something? Start a Discussion or comment below!

**Comment by Gwaspac** [comment]
The first day that I find my self here I strongly believe that all my mistake i will correct them

**Reply by 58580664** [reply]
Yes I will

**Comment by 58580664** [comment]
@queenofcorgis

**Comment by 81CuongVn** [comment]
niice i try to hard receive them

**Comment by JABEALE** [comment]
I wanted to know if i can download grithub on oppo Android phone

**Reply by Kaburu12** [reply]
haha is it possible?

**Reply by iamrkaran** [reply]
> I wanted to know if i can download GitHub on oppo Android phone

its possible  to install GitHub on mobile for issues community watching repo..

**Comment by Kaburu12** [comment]
Hello guys , I'm glad to be here. I wanna make friends

**Reply by righteousness1** [reply]
Hello, you can do it.

**Reply by aqib-m31** [reply]
Hey! 👋😄

**Comment by MamaT-Superwoman** [comment]
New to GitHub

**Comment by aqib-m31** [comment]
I'm new to GitHub Community. I'm here to learn things and get to know the community members and make friends. 😄

**Reply by Devilishhh** [reply]
Hello and welcome!! I am also new to the GitHub Community, I am always open to making new connections
